en/chromebook_setup/instructions.md

```diff
@@ -37,29 +44,40 @@ stuff useful computer code into for a project we're working on. We use them to
 keep the various bits of code we want for our various projects separate so
 things don't get mixed up between projects.
 
-In your terminal at the bottom of the Cloud 9 interface, run the following:
+In your terminal, type the following lines. Hit enter after each one to send the
+instruction to the computer.
 
-{% filename %}Cloud 9{% endfilename %}
+{% filename %}CodeAnywhere{% endfilename %}
 ```
 sudo apt update
-sudo apt install python3.6-venv
+sudo apt install python3.4-venv
 ```
 
 If this still doesn't work, ask your coach for some help.
 
 Next, run:
 
-{% filename %}Cloud 9{% endfilename %}
+{% filename %}CodeAnywhere{% endfilename %}
 ```
 mkdir djangogirls
 cd djangogirls
-python3.6 -mvenv myvenv
+python3 -mvenv myvenv
 source myvenv/bin/activate
+pip install --upgrade pip
 pip install django~=1.11.0
 ```
 
 (note that on the last line we use a tilde followed by an equal sign: ~=).
 
+Finally, right click on "tutorial-container" in the sidebar and select
+_Refresh_. You should see a new folder, "djangogirls", appear in the list in the
+sidebar.
+
+### Golden Rule (of Chromebooks)
+
+Throughout the tutorial, if you see instructions for OS X, Linux, or Windows but
+not specifically for Chromebooks, follow whatever the Linux instructions are.
```

Comment:
I've seen quite a few people following the Chromebook instructions even though they didn't have Chromebooks – for good reasons or just because they followed the wrong path in the install step. To address such cases, I'd suggest referring to "CodeAnywhere" instead of Chromebooks so that anyone using that knows what commands to refer to.